releasenotes/notes/circuit-dag-converters-a21ab8a7d344acbb.yaml

```
@@ -0,0 +1,5 @@
+---
+features_c:
+  - |
+    Added the :cpp:func:`qk_dag_to_circuit` and :cpp:func:`qk_circuit_to_dag` functions
```

```suggestion
    Added the :c:func:`qk_dag_to_circuit` and :c:func:`qk_circuit_to_dag` functions
```

Ray just fixed this in #15250 🎉